iib/web/models.py

```diff
@@ -350,6 +362,15 @@ def _state_to_json(state):
 
         return rv
 
+    def get_modifiable_keys(self):
```

Comment:
Optional: how about renaming `get_modifiable_keys` to `get_mutable_keys`?

Reply:
Great suggestion! Done.